ipatests/test_install/test_changeconf.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+
+from __future__ import absolute_import
+
+import pytest
+from ipaclient.install.ipachangeconf import IPAChangeConf
+
+
+@pytest.fixture(scope='session')
+def config_file(tmpdir_factory):
+    file = tmpdir_factory.mktemp('data').join('config_file.conf')
+    file.write('SOME_CONF /some/user/defined/path\n')
+    return file
+
+
+def test_addifnotset_action(config_file):
+    """Test if addifnotset action adds a comment about the modified conf.
+
+    IPA don't want to break existing configuration, if a value already exists
+    it adds a comment to the modified setting and a note about that on the line
+    above.
+
+    New settings will be added without any note.
+    """
+    ipa_conf = IPAChangeConf('IPA Installer Test')
+    ipa_conf.setOptionAssignment(' ')
+
+    opts = [
+        {
+            'action': 'addifnotset',
+            'name': 'SOME_CONF',
+            'type': 'option',
+            'value': '/path/defined/by/ipa',
+        },
+        {
+            'action': 'addifnotset',
+            'name': 'NEW_CONF',
+            'type': 'option',
+            'value': '/path/to/somewhere',
+        },
+    ]
+
+    ipa_conf.changeConf(str(config_file), opts)
+
+    assert config_file.readlines() == [
+        '#SOME_CONF modified by IPA\n',
```

Comment:
``#text`` is hard to read. Could you change the code to emit ``# text`` instead?

Reply:
Agreed.